gen-dao/src/main/java/fr/gaulupeau/apps/Poche/GenerateDao.java

```diff
@@ -17,14 +17,23 @@ public static void main(String[] args) throws Exception {
         article.addStringProperty("url").columnName("url");
         article.addBooleanProperty("favorite").columnName("favorite");
         article.addBooleanProperty("archive").columnName("archive");
-        article.addBooleanProperty("sync").columnName("sync");
-        article.addDateProperty("updateDate").columnName("update_date");
+        article.addBooleanProperty("sync").columnName("sync"); // TODO: remove
+        article.addDateProperty("updateDate").columnName("update_date"); // TODO: check: what is this prop for?
```

Comment:
I think it was to display somehow the date the article had been added through v1's RSS feeds. Not relevant until we use the API from v2.